src/components/countDown/CountDown.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { useRef,useState,useEffect } from 'react';
```

Comment:
1. It is not necessary to import react in every file, in any one file would do.
2. In case you're keeping it, you could write line 2 in line 1 itself as the source is the same.